src/cli/summarize.py

```diff
@@ -116,44 +118,43 @@ def convert_to_pandas(self, data: Dict[str, Dict[Any, Any]]) -> DataFrame:
         df = pd.concat({key: DataFrame(value).T for key, value in data.items()})
         return df.rename_axis(["dir", "test"])
 
-    def save_mean_data(self, mean_of_dir: DataFrame, src_dirs: List[str], out_dir: Path) -> None:
+    def save_mean_data(self, mean_of_dir: DataFrame, src_dirs: List[Path], out_dir: Path) -> None:
         out_dir.mkdir(parents=True, exist_ok=True)
         with open(out_dir.joinpath(self.filename_out_data), "w") as f:
             # Use pandas to print data to file
             f.write("Summarized data:\n")
             f.write(mean_of_dir.to_string())
             f.write("\n\n")
             for src_dir in src_dirs:
-                f.write(f"dir: {src_dir}\n")
-                f.write(src_dir)
+                f.write(f"dir: {str(src_dir)}\n")
```

Comment:
I think `str` function is unnecessary in f-string

Reply:
fixed